.github/workflows/edge.yml

```diff
@@ -0,0 +1,64 @@
+name: Mondoo Docker Tag Edge
```

Comment:
maybe: Release Mondoo Edge Container Image

Reply:
done and also renamed the other workflow so the names are consistent between edge and actual release